exercises/tasks/task_05.py

```diff
@@ -3,9 +3,8 @@
 
 # Searching for a connection displays results
 def test_searching_for_connection_displays_results(page):
-    # 1. Open the kiwi.com website (wait for page to load)
+    # 1. Open the kiwi.com website (wait for page to load) and accept cookies
     kiwi_page = KiwiPage(page)
-    kiwi_page.open_kiwi_website()
```

Comment:
This row was placed here intentionally in order for the attendees to have a working example of how to call methods of a testing class (it's a new concept not shown in the previous tasks by the time this one is being done).